fitbenchmarking/systests/expected_results/minimizers.txt

```diff
@@ -0,0 +1,17 @@
+                                     dfogn                   BFGS                     gn                 amoeba       lm-scipy
```

Comment:
We now have pygsl merged in so we probably should pull out a minimizer from there? Also, I don't see the `minuit` minimizer

Reply:
These get pulled in automatically, so when I fix merge conflicts they'll be added